apps/worker/tasks/upload_processor.py

```diff
@@ -117,6 +120,24 @@ def on_processing_error(error: ProcessingError):
                     error=Errors.INTERNAL_RETRYING,
                 )
                 self.retry(max_retries=MAX_RETRIES, countdown=FIRST_RETRY_DELAY)
+            elif (
+                error.is_retryable
+                and self.request.retries >= MAX_FILE_NOT_FOUND_RETRIES
+            ):
+                sentry_sdk.capture_exception(
+                    FileNotInStorageError(
+                        f"File not found in {error.params.get('location')} for upload {arguments.get('upload_id')}"
```

Comment:
Thoughts on making the error generic and the parameters dynamic?

If we consider something like "File not found in storage" vs "File not found in <dynamic val> for upload <dynamic val>", the 1st one will show all the logs for this type of error, whereas, _I think_, the 2nd one will show as maany entries in sentry. 

If this behaves how I think, I'd prefer to lower the cardinality of the error (1st approach) and provide the details solely in the context, as you are doing rn